tests/cpp_unit_tests/test_y_bus.cpp

```diff
@@ -276,6 +281,171 @@ TEST_CASE("Test fill-in y bus") {
     CHECK(ybus.map_lu_y_bus == map_lu_y_bus);
 }
 
+TEST_CASE("Incremental update y-bus") {
+    /*
```

Comment:
please do not use `/* ... */` because it has the same issues as C-style MACROS. In addition, it has different formatting behavior for different editors.

The only exception is the very specific case of inline and in-code comments, e.g.
```cpp
inline void function(Foo /* unused_foo */, Bar bar) {
  baz(bar);
}
```